src/components/_app/KeyPairProvider.tsx

```diff
@@ -265,6 +266,7 @@ const KeyPairProvider = ({ children }: PropsWithChildren<unknown>): JSX.Element
               "You've connected your account from a new device, so you have to sign a new message to stay logged in",
             duration: 5000,
           })
+          openWalletSelectorModal()
```

Comment:
You showed this IRL today, is it still needed, knowing there are cases when this doesn't run, but the modal is needed (when there is no previous keypair, which has been invalidated)

Reply:
Nice catch, we don't need this line anymore! Fixed in: https://github.com/agoraxyz/guild.xyz/pull/950/commits/0e2d3b4aa9a7b3fc8b5da47c0c0a78aad058d424